### ISSUE TITLE
Upgrade Swagger UI and use new standalone version

### ISSUE THREAD

**Issue by duffn** [ruby-grape/grape-swagger-rails#106]
This could use the simplified HTML/JS/CSS version of Swagger UI.

- Migrate to this version https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/installation.md#plain-old-htmlcssjs-standalone
	- This is an example only. Need to pull down the assets into the repo.
- Update Rake task to pull new versions as necessary
- Update Swagger `index.html.erb` as necessary

I'd also like to:

- Update the dummy app in this style of 6.0+ if not already
- Add a test matrix for newer versions of `grape-swagger`, Ruby, and Rails

I'm trying to figure out the best order to do all of these.

**Comment by dblock** [comment]
This project needs some love. You probably know best in which order to start with. I typically begin CI, so would do the "also like to" part first.

**Comment by duffn** [comment]
Agreed, here's a first pass at the test matrix: https://github.com/ruby-grape/grape-swagger-rails/pull/107

**Comment by moskvin** [comment]
This is now done in `v1.0.0`.

The project has moved to the standalone Swagger UI integration and now ships the maintained Swagger UI v5 bundle/preset assets. The update task and frontend wiring were also refreshed accordingly.

Closing as resolved.
